projects/commudle-admin/src/app/app.component.scss

```diff
@@ -32,9 +32,13 @@
           margin-left: $space-16;
 
           .menu-user-avatar {
-            width: 45px;
-            height: 45px;
-          }
+              max-width: 40px;
+              max-height: 40px;
+              img{
+                  width: 100%;
```

Comment:
Also please take care that the code you write is formatted

Reply:
Sure!